src/components/checkbox/__tests__/CheckboxField-test.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import React from 'react';
+import CheckboxField from '../CheckboxField';
+
+describe('components/checkbox/CheckboxField', () => {
+    const getWrapper = (props = {}) => shallow(<CheckboxField {...props} />);
+
+    test('should render properly when turned on', () => {
```

Comment:
Might be worthwhile to use `test.each` for these, since the value of `value` is the only difference.